tests/unit/test_logging_default_config.py

```diff
@@ -31,3 +34,21 @@ def test_enable_response_logging(switch):
 def test_enable_response_body_logging(switch):
     enable_response_body_logging(switch)
     assert config.is_response_body_logging_enabled() == switch
+
+
+@pytest.mark.parametrize("switch", [10, 20, 30, 40, 50])
+def test_set_logging_level(switch):
+    set_logging_level(switch)
+    assert config.get_logging_level() == switch
+
+
+@pytest.mark.parametrize("switch", ["INFO", "DEBUG", "WARNING", "ERROR", "CRITICAL"])
+def test_set_logging_level_string(switch):
+    set_logging_level(switch)
+    assert config.get_logging_level() == level_map[switch]
+
+
+@pytest.mark.parametrize("switch", ["INVALID", "INFODEBUG", 60, 11, 35, 45, 11010210120201, None])
```

Comment:
**praise**: Great initiative for handling the sad path scenarios. This is why I am suggesting refactoring to an enumeration type instead, so we ensure the user won’t end up using the wrong log level due to typos (i.e., validated at runtime or at type-checking time if using a type checker)